pkg/storage/tpr/install_types.go

```diff
@@ -17,11 +17,13 @@ limitations under the License.
 package tpr
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
-	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
+	tprs_v1beta1 "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5/typed/extensions/v1beta1"
```

Comment:
@simonleung8 can you choose either the aforementioned alias or use something else without underscores?